Make.config

```diff
@@ -98,18 +98,16 @@ MIN_OSX_BUILD_VERSION=10.14
 MIN_OSX_VERSION_FOR_IOS=10.11
 MIN_OSX_VERSION_FOR_MAC=10.11
 
+# Make sure that the following does match the sdk that is preset in Xcode. Else we will have issues with mtouch (See https://github.com/xamarin/xamarin-macios/issues/7705)
 IOS_SDK_VERSION=13.2
 # When bumping OSX_SDK_VERSION also update the macOS version where we execute on bots in jenkins/Jenkinsfile (in the 'node' element)
 OSX_SDK_VERSION=10.15
 WATCH_SDK_VERSION=6.1
 TVOS_SDK_VERSION=13.2
 
-# If the max OS version does not match the SDK version (for instance the iOS
-# 12.2 SDK supported both iOS 12.3 and iOS 12.4), then these variables can be
-# set to something other than the corresponding SDK versions.
-MAX_IOS_VERSION=13.3
+MAX_IOS_VERSION=13.2
```

Comment:
We should get rid of that completely. The version check in `builds/Makefile` should be reverted to check against `IOS_SDK_VERSION`. See https://github.com/xamarin/xamarin-macios/commit/aac92a1c20385c167a961774bd32ba0dd3601cbb

Reply:
So given https://github.com/xamarin/xamarin-macios/commit/806404fa51886aa1c066a7271a5493e4ef08c97e, I suggest we just change those variables to `MAX_IOS_SIMULATOR_VERSION` (same for WATCHOS, TVOS).

This would avoid confusion and we still need to change `versions-check.csharp` to check against the SDK version variables (: